CMake/External_OpenCV.cmake

```diff
@@ -200,6 +236,11 @@ if (CMAKE_COMPILER_IS_GNUCC AND NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 6)
   list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DENABLE_PRECOMPILED_HEADERS:BOOL=OFF)
 endif()
 
+# OpenCV 3.3.0 has an option to enable C++ 11
+if (fletch_BUILD_CXX11)
+  list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DENABLE_CXX11:BOOL=ON)
```

Comment:
Doesn't OpenCV work with the CMAKE_CXX_STANDARD:STRING=11? We already pass that information into the configuration. If it doesn't then this is fine. If it does, it would be good to use the internal CMake way.